### ISSUE TITLE
Menu bar in GUI is invisible

### ISSUE THREAD

**Issue by Mengya-W** [MouseLand/suite2p#1142]
### Describe the issue:

I installed suite2p on a new Windows computer, but the menu bar is invisible in the GUI (1st image). It only appears when the suite2p GUI window is inactive, such as when I activate another window, e.g. Zoom (2nd image). I've tried restarting the program and computer, as well as reinstalling Anaconda and Suite2p, but the issue persists. Could you please advise on how to resolve this? Thank you!

![Screen Shot 2024-08-22 at 6 44 43 PM](https://github.com/user-attachments/assets/35ae3de7-0f56-4246-bfa6-119516a3edcd)
![Screen Shot 2024-08-22 at 6 45 06 PM](https://github.com/user-attachments/assets/052c7c54-9ca7-4218-8faf-29b46fc751dc)

### Provide environment info:

 active environment : suite2p
    active env location : C:\Users\wangm\anaconda3\envs\suite2p
            shell level : 2
       user config file : C:\Users\wangm\.condarc
 populated config files : C:\Users\wangm\.condarc
          conda version : 24.5.0
    conda-build version : 24.5.1
         python version : 3.12.4.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=x86_64_v4
                          __conda=24.5.0=0
                          __cuda=12.5=0
                          __win=0=0
       base environment : C:\Users\wangm\anaconda3  (writable)
      conda av data dir : C:\Users\wangm\anaconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\wangm\anaconda3\pkgs
                          C:\Users\wangm\.conda\pkgs
                          C:\Users\wangm\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\wangm\anaconda3\envs
                          C:\Users\wangm\.conda\envs
                          C:\Users\wangm\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/24.5.0 requests/2.32.2 CPython/3.12.4 Windows/11 Windows/10.0.22631 solver/libmamba conda-libmamba-solver/24.1.0 libmambapy/1.5.8 aau/0.4.4 c/. s/. e/.
          administrator : False
             netrc file : None
           offline mode : False

**Comment by pjdavis13** [comment]
I'm having the same issue. Were you ever able to find a resolution?

**Comment by Rebecca-SJP** [comment]
I'm having the same issue, too, in Windows 11

**Comment by liuliuzhizhi** [comment]
I'm having the same issue. Were you ever able to find a resolution?

**Comment by Mengya-W** [comment]
No, but the functions seem to be working, so I’m just going with it.

**Comment by huda-github-20** [comment]
I'm having the same issue. Has anyone found a solution to this?
